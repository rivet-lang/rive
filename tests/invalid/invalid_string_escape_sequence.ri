func main() {
    _ = "abc\_def";
}
