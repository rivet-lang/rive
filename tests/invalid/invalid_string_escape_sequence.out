tests/invalid/invalid_string_escape_sequence.ri:2:14: error: unknown escape sequence `_`
    2 |     _ = "abc\_def";
      |              ^
rivet: error: could not compile module `invalid_string_escape_sequence`, aborting due to previous error